src/test/java/com/fasterxml/jackson/databind/BaseMapTest.java

```diff
@@ -292,9 +292,18 @@ protected void assertStandardEquals(Object o)
     /**********************************************************
      */
 
-    @SuppressWarnings("unchecked")
     protected Map<String,Object> writeAndMap(ObjectMapper m, Object value)
         throws IOException
+    {
+        return writeAndMap2(m, value);
+    }
+
+    /**
+     * To prepare for 3.0 that will move away from JUnit 4, we need to avoid using protected methods.
+     */
+    @SuppressWarnings("unchecked")
+    public static Map<String,Object> writeAndMap2(ObjectMapper m, Object value)
```

Comment:
This works, but I'd also recommend also checking if we could and should just get rid of this method.
I _think_ it's old one and predates existence of `writeValueAsString()` so was more useful before that.

Reply:
Ok, that's called from quite a few places. Hmmh.

Also: for new, non-inheritance based test classes, maybe we should migrate methods into `TestUtil` class with static methods, and essentially start getting rid of `BaseXxx` classes for tests.
During transition `BaseMapTest` could just call methods in `TestUtil` (or maybe it'd be `DatabindTestUtil`?)

Reply:
Sounds great, added so! 👍🏼